src/Nest/XPack/MachineLearning/Job/Results/Bucket.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Nest
+{
+	public class Bucket
+	{
+		[JsonProperty("job_id")]
```

Comment:
This PR is full of awesome xmldocs but this one is left out 😿

Reply:
Done.